backend/app/Models/Choice.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Choice extends Model
+{
+    use HasFactory, SoftDeletes;
+    public function word()
+    {
+        return $this->belongsTo(WordQuestion::class,'word_id','id');
```

Comment:
@rogeliojohnoliverio no need to indicate the relationship since it's already set in the migration. Please recheck all your models and update it.
```suggestion
        return $this->belongsTo(WordQuestion::class);
```